File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/pfes/PFE0101U00GrdDCodeInfo.java
package nta.med.data.model.ihis.pfes;

public class PFE0101U00GrdDCodeInfo {
    private String codeType    ;
    private String code         ;
    private String codeName    ;
    private String codeNameRe ;
    private String codeValue   ;
	public PFE0101U00GrdDCodeInfo(String codeType, String code,
			String codeName, String codeNameRe, String codeValue) {
		super();
		this.codeType = codeType;
		this.code = code;
		this.codeName = codeName;
		this.codeNameRe = codeNameRe;
		this.codeValue = codeValue;
	}
	public String getCodeType() {
		return codeType;
	}
	public void setCodeType(String codeType) {
		this.codeType = codeType;
	}
	public String getCode() {
		return code;
	}
	public void setCode(String code) {
		this.code = code;
	}
	public String getCodeName() {
		return codeName;
	}
	public void setCodeName(String codeName) {
		this.codeName = codeName;
	}
	public String getCodeNameRe() {
		return codeNameRe;
	}
	public void setCodeNameRe(String codeNameRe) {
		this.codeNameRe = codeNameRe;
	}
	public String getCodeValue() {
		return codeValue;
	}
	public void setCodeValue(String codeValue) {
		this.codeValue = codeValue;
	}
    

}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Pfes/PFE0101U00GrdDCodeInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Pfes
{
    [Serializable]
	public class PFE0101U00GrdDCodeInfo
	{
		private String _codeType;
		private String _code;
		private String _codeName;
		private String _codeNameRe;
		private String _codeValue;

		public String CodeType
		{
			get { return this._codeType; }
			set { this._codeType = value; }
		}

		public String Code
		{
			get { return this._code; }
			set { this._code = value; }
		}

		public String CodeName
		{
			get { return this._codeName; }
			set { this._codeName = value; }
		}

		public String CodeNameRe
		{
			get { return this._codeNameRe; }
			set { this._codeNameRe = value; }
		}

		public String CodeValue
		{
			get { return this._codeValue; }
			set { this._codeValue = value; }
		}

		public PFE0101U00GrdDCodeInfo() { }

		public PFE0101U00GrdDCodeInfo(String codeType, String code, String codeName, String codeNameRe, String codeValue)
		{
			this._codeType = codeType;
			this._code = code;
			this._codeName = codeName;
			this._codeNameRe = codeNameRe;
			this._codeValue = codeValue;
		}

	}
}